tests/test_pipeline.py

```diff
@@ -159,16 +159,3 @@ def test_invalid_argument(default_pipeline_args, invalid_component_args, tmp_pat
 
     with pytest.raises((ValueError, TypeError)):
         pipeline.compile()
-
-
-def test_reusable_component_op():
```

Comment:
I would not delete this test. Could you implement the suggestion by @RobbeSneyders for linking the reusable pipelines with a temporary symlink

Reply:
Yes for the moment I reverted this in order to test out the entire pipeline.